CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## 18.2.0 - 2021-10-29
+This release includes several bug fixes, introduces Klarna as a payment method binding, and renables wechat pay within the SDK
+* [4323](https://github.com/stripe/stripe-android/pull/4323) reship wechat module 
+* [4325](https://github.com/stripe/stripe-android/pull/4325) Add klarna to sdk w/ example
```

Comment:
35d5b367ac2e2f3ac68f7f9683afc3528b9b67f0 Bump tensorflow-lite from 2.4.0 to 2.6.0 (#4339)
0604923a9da50c9b1f3bdaeda2d581d5f487dfef Bump okio from 2.10.0 to 3.0.0 (#4340)
a5476e9d33a10d724ea64ff204b809b14462ce26 Bank value is allowed to be null in the case
6eec0cfd4ec96c5e520641bfcdc47717adf78e16 Bump lifecycle-viewmodel-compose from 2.4.0-rc01 to 2.4.0 (#4330)
c0336b1c7dc300b6ea6aff0814595b84cdbf8927 Bump daggerVersion from 2.39.1 to 2.40 (#4329)
148ad3363e1feb0ef15e43ab0dd7ce7291dca301 (refs/bisect/bad) Card number, CVC, postal, and expiration date should only show digits in keypad (#4309)
705390657782b2ce820304e92a536105c774db12 (refs/bisect/good-705390657782b2ce820304e92a536105c774db12) Bump lifecycle-viewmodel-compose from 1.0.0-alpha07 to 2.4.0-rc01 (#4198)
790d0df9a240e94a630e62b7b8adb9429ec59c34 When processing Result for a PI, refresh until reaches deterministic state (#4296)
aba2e81b3a9b8fff84b38cf1e7057751f13ad9f7 (refs/bisect/good-aba2e81b3a9b8fff84b38cf1e7057751f13ad9f7) Bump composeVersion from 1.0.2 to 1.0.4 (#4290)